pypit/data/settings/settings.armlsd

```diff
@@ -3,9 +3,9 @@
 # RUNNING ARMLSD
 
 # REDUCTION RULES
-reduce calibrate nonlinear True          # Perform a non-linear correction
-reduce calibrate flux True       # Perform a flux calibration
-reduce flexure method boxcar      # Perform flexure correction on objects using boxcar extraction.  Options are: None, boxcar, slitcen [this performs flexure correction before extraction of objects]
+reduce calibrate nonlinear True # Perform a non-linear correction
```

Comment:
I think you want to keep this option set to False. True won't do anything yet. The intention of this command is to perform a nonlinear correction when the detector counts doesn't increase linearly with exposure time.